gradle/libs.versions.toml

```diff
@@ -0,0 +1,73 @@
+[versions]
+antlr4 = "4.11.1"
+checkstyle = "10.3.4"
+errorprone = "2.15.0"
+googleJavaFormat = "1.15.0"
+grpc = "1.49.1"
+guava = "31.1-jre"
+immutables = "2.9.2"
+jacoco = "0.8.8"
+jandex = "2.4.3.Final"
+jandexPlugin = "1.82"
+jmh = "1.35"
+junit = "5.9.1"
+nessieBuildPlugins = "0.2.12"
+nessieRunPlugin = "0.27.3"
+protobuf = "3.21.6"
+shadowPlugin = "7.1.2"
+slf4j = "1.7.36"
+spotlessPlugin = "6.11.0"
+
+[bundles]
+# Bundles serve two purposes:
+# 1. Group dependencies together for renovatebot, which cares about bundles. Those bundles, that
+#    _solely_ exist to for dependency management via renovatebot, start with `managed-`.
+# 2. Bundles used in build files to group dependencies that are commonly used together.
+managed-grpc = ["grpc-protobuf", "grpc-stub", "grpc-netty-shaded"]
+managed-immutables = ["immutables-value-annotations", "immutables-value-processor"]
+managed-jmh = ["jmh-core", "jmh-generator-annprocess"]
+
+junit-testing = ["assertj-core", "junit-jupiter-api", "junit-jupiter-params"]
+
+[libraries]
+agrona = { module = "org.agrona:agrona", version = "1.17.1" }
+antlr-antlr4 = { module = "org.antlr:antlr4", version.ref = "antlr4" }
+antlr-antlr4-runtime = { module = "org.antlr:antlr4-runtime", version.ref = "antlr4" }
+assertj-core = { module = "org.assertj:assertj-core", version = "3.23.1" }
+findbugs-jsr305 = { module = "com.google.code.findbugs:jsr305", version = "3.0.2" }
+google-java-format = { module = "com.google.googlejavaformat:google-java-format", version.ref = "googleJavaFormat" }
+grpc-protobuf = { module = "io.grpc:grpc-protobuf", version.ref = "grpc" }
+grpc-stub = { module = "io.grpc:grpc-stub", version.ref = "grpc" }
+grpc-netty-shaded = { module = "io.grpc:grpc-netty-shaded", version.ref = "grpc" }
```

Comment:
Just wondering why Netty is required as a direct dependency for a CEL implementation? :thinking:

Reply:
Only for the conformance tests.